coralproject/talk
# RTE Bugfixes

## What does this PR do?
- https://www.pivotaltracker.com/story/show/157607216
- https://www.pivotaltracker.com/story/show/157434885
- https://www.pivotaltracker.com/story/show/157641632

## How do I test this PR?

- Make sure focus is not on the RTE and then click on e.g. the bold button (Should work in IOS and desktop browsers)
- Try making quotes out of different selections with multiple paragraph, it should work as expected
- Quote an empty line, and unquote it immediately, the previous empty line should reappear
